### DISCUSSION TITLE
Manual keybind display?

### DISCUSSION THREAD

**Discussion by handsupdb** [Hekili/hekili#960] [Q&A] [answered]
I'm wondering if there's a way to manually set the keybind that displays with an ability in the priority list. So rather than dynamically updating the keybind based on where the ability is on the bar - it just displays a static text on the ability that I set.

Background:
I want to macro certain secondary functions to some keybinds (i.e trinket/racial/pet ability) so that it's getting hit anyway.
A prime example is as Unholy DK adding pet attack onto other abilities to make pet switch/attach faster, or instructing gnaw/petmoveto along with Dark Transformation.

In the Dark Transformation example Hekili will display the DT ability no problem, but because DT itself doesn't have a keybind (the macro is what's on the action bar) it doesn't display the keybind.

I don't "need" this functionality - of course if I see DT icon pop up I know what key is DT.
Just thought it'd be nice for in the future if I use Hekili as a learning tool for a different class rather than as the "you screwed up your rotation/overlap, look here to get back on track" that I currently do.

**Comment by Hekili** [comment, marked as answer]
/hekili > Abilities > Override Keybind Text

**Reply by handsupdb** [reply]
You sir, are a gentleman and a scholar.
And I am an unobservant dunce.

Thank you!
